test/gtest/uct/ib/test_ib_md.cc

```diff
@@ -171,6 +171,41 @@ UCS_TEST_P(test_ib_md, ib_md_umr_direct, "REG_METHODS=direct") {
     ib_md_umr_check(&rkey_buffer[0], false);
 }
 
+UCS_TEST_SKIP_COND_P(test_ib_md, explicit_odp_reg_advise,
+                     !check_caps(UCT_MD_FLAG_REG | UCT_MD_FLAG_ADVISE),
+                     "MLX5_DEVX_OBJECTS=dct,dcsrq")
+{
+    test_reg_advise(UCS_KBYTE, UCS_KBYTE);
+}
```

Comment:
what does it check?

Reply:
During this test we just test memory registration + advise for the ODP. I do not know do we have guarantee that after advise for the ODP memory region pinned memory won't increase so there is no check.

But maybe it would be better to do memory registration, then check the pinned memory and then call the advise verb. WDYT?

Reply:
I overthought that and decided to move all the tests to the `test_md.cc` file since it seems more about all the MDs now, not only about the IB. Pls review.